src/examples/table/TableVirtualized.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { FixedSizeList } from 'react-window';
+import { Table, Head, HeaderRow, HeaderCell, Body, Row, Cell } from '@zendeskgarden/react-tables';
+
+const rowData: {
+  id: number;
+  subject: string;
+  requester: string;
+  requested: string;
+  type: string;
+}[] = [];
+
+for (let x = 1; x <= 100000; x++) {
+  rowData.push({
+    id: x,
+    subject: 'Example subject',
+    requester: 'John Doe',
+    requested: '15 minutes ago',
+    type: 'Ticket'
+  });
+}
+
+/* stylelint-disable declaration-no-important */
+const ScrollableTable = styled(Table).attrs({ role: 'presentation' })`
+  display: block !important;
+`;
+
+const ScrollableHead = styled(Head)`
+  display: block;
+`;
+
+const ScrollableHeaderRow = styled(HeaderRow).attrs({ role: 'row' })`
+  display: table !important;
+  width: 100%;
+  table-layout: fixed;
+`;
+
+const ScrollableHeaderCell = styled(HeaderCell).attrs({ role: 'columnheader' })``;
+
+const ScrollableBody = styled(Body)`
+  display: block !important;
+`;
+
+const ScrollableRow = styled(Row).attrs({ role: 'row' })`
+  display: table !important;
+  table-layout: fixed;
+`;
+
+const ScrollableCell = styled(Cell).attrs({ role: 'cell' })``;
+/* stylelint-enable declaration-no-important */
+
```

Comment:
There seems to be a glitch with "View code" for this example. Might have something to do with the `stylelint-disable/enable` comments? IDK. Either way, I'd recommend scoping the `declaration-no-important` rule if possible (similar to the way `overrides` are applied in .eslintrc.json).

Reply:
With this using stylelint the scoping options are pretty limited. Rather than turning it off globally I've moved the `stylelint-disable` commands within the styled element. This seems to have corrected the `View Code` styling.

Reply:
@jzempel what do you think of the updated stylelint comments?